pontoon/base/management/commands/update_auth_providers.py

```diff
@@ -0,0 +1,70 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import
+
+from django.core.exceptions import ObjectDoesNotExist
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from django.contrib.sites.models import Site
+from allauth.socialaccount.models import SocialApp
+from allauth.socialaccount.providers.fxa.provider import FirefoxAccountsProvider
+from allauth.socialaccount.providers.github.provider import GithubProvider
+
+
+FXA_PROVIDER_ID = FirefoxAccountsProvider.id
+GITHUB_PROVIDER_ID = GithubProvider.id
+
+
+class Command(BaseCommand):
+    help = ('Ensures an allauth application exists and has credentials that match settings')
+
+    def update_provider(self, data):
+        try:
+            # Update the existing provider with current settings.
+            app = SocialApp.objects.get(provider=data['provider'])
+            self.stdout.write("Updating existing authentication provider (pk=%s)" % app.pk)
```

Comment:
I would move this to the bottom, after `app.save()` - to follow the patter in the `except` block.